### PR TITLE
Discochuck adding glcr prices usd

### PR DESCRIPTION
**Pull request by discochuck**
Brief comments on the purpose of your changes:

**For Dune Engine V2**

I've checked that:

### General checks:
* [ ] I tested the query on dune.com after compiling the model with dbt compile (compiled queries are written to the target directory)
* [ ] I used "refs" to reference other models in this repo and "sources" to reference raw or decoded tables 
* [ ] if adding a new model, I added a test
* [ ] the filename is unique and ends with .sql
* [ ] each sql file is a select statement and has only one view, table or function defined  
* [ ] column names are `lowercase_snake_cased`
* [ ] if adding a new model, I edited the dbt project YAML file with new directory path for both models and seeds (if applicable)
* [ ] if wanting to expose a model in the UI (Dune data explorer), I added a post-hook in the JINJA config to add metadata (blockchains, sector/project, name and contributor Dune usernames)

### Pricing checks:
* [ ] `coin_id` represents the ID of the coin on coinpaprika.com
* [ ] all the coins are active on coinpaprika.com (please remove inactive ones)

### Join logic:
* [ ] if joining to base table (i.e. ethereum transactions or traces), I looked to make it an inner join if possible

### Incremental logic:
* [ ] I used is_incremental & not is_incremental jinja block filters on both base tables and decoded tables
  * [ ] where block_time >= date_trunc("day", now() - interval '1 week')
* [ ] if joining to base table (i.e. ethereum transactions or traces), I applied join condition where block_time >= date_trunc("day", now() - interval '1 week')
* [ ] if joining to prices view, I applied join condition where minute >= date_trunc("day", now() - interval '1 week')

### CODE REVIEW

File: models/prices/avalanche_c/prices_avalanche_c_tokens.sql [hunk 1]
```diff
@@ -17,6 +17,7 @@ FROM
     VALUES
 
     ("aave-new","avalanche_c","AAVE.e","0x63a72806098bd3d9520cc43356dd78afe5d386d9",18),
+    ("glcr-glacier","avalanche_c","GLCR","0x3712871408a829C5cd4e86DA1f4CE727eFCD28F6",18),
```
Comment (jeff-dude) [inline models/prices/avalanche_c/prices_avalanche_c_tokens.sql:20]: please note this id value isn't found on the API: https://api.coinpaprika.com/v1/coins/glcr-glacier

Comment (jeff-dude) [reply models/prices/avalanche_c/prices_avalanche_c_tokens.sql:20]: could it be a different value?

**Review by jeff-dude — CHANGES_REQUESTED** [review]

File: models/prices/avalanche_c/prices_avalanche_c_tokens.sql (hunk 1 above)
Comment (discochuck) [reply models/prices/avalanche_c/prices_avalanche_c_tokens.sql:20]: sorry, got a bit ahead of my self. didn't realize the coincaprika listing was a prerequisite.

I'll check back post listing (apparently about a month+)

Comment (jeff-dude) [reply models/prices/avalanche_c/prices_avalanche_c_tokens.sql:20]: okay, will close PR for now, either reopen or start a new one when ready 🤝

**Pull request closed without merging**